extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -0,0 +1,15 @@
+amp-story-audio-sticker {
+  overflow: hidden !important;
```

Comment:
Did you want to have a maximum width and maximum height for the sticker? Eg: `max-width: calc(var(--story-vw) * 20)`? (I'm making up the numbers here)

Reply:
Based on the UX requirements, there will be only two options of size: 120 x 120 or 180 x 180px. Developers would need to choose either one, e.g. `<amp-story-audio-sticker size='large'></amp-story-audio-sticker>`. I will make this change in the next PR.